cli/cmd/stop.go

```diff
@@ -32,9 +36,42 @@ func NewStopCmd() *cobra.Command {
 		},
 	}
 
+	stopCmd.Flags().BoolVarP(&autoYes, "yes", "y", false,
+		`Automatic yes to confirmation prompt`)
+
 	return stopCmd
 }
 
+func internalStopWithConfirmationModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
+	if !isConfigExist(cmdCtx) {
+		return errNoConfig
+	}
+
+	if !autoYes {
```

Comment:
Please, also check for NoPrompt flag here.

Reply:
Now there is a check for --no-prompt